internal/offset/core_test.go

```diff
@@ -112,3 +131,20 @@ func TestCore_DeleteOffsetStatus(t *testing.T) {
 	assert.NoError(t, err)
 	assert.NotNil(t, oStatus)
 }
+
+func TestCore_RollBackOffset(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	mockRepo := mocks.NewMockIRepo(ctrl)
+	core := NewCore(mockRepo)
+	offset := getDummyOffsetModel()
+	ctx := context.Background()
+	mockRepo.EXPECT().Exists(gomock.Any(), offset.Key()).Return(true, nil)
+	mockRepo.EXPECT().Get(gomock.Any(), offset.Key(), offset).DoAndReturn(
```

Comment:
use `AssignableToTypeOf(context.Context)` instead of `Any()` for ctx

Reply:
context created in the actual function call is different from interface context. So cannot use AssignableToTypeOf here